### PR TITLE
create tests for buy_asset_from_market and ckrrected other tests

### PR DESCRIPTION
**Pull request by Lex-23**

### CODE REVIEW

**Comment by Lex-23** [comment]
[Write logic for BUY ASSETS from market](https://trello.com/c/nzNYUyZA/16-write-logic-for-buy-assets-from-market)

File: cryptostock/market/tests/test_buy_asset_from_market.py [hunk 1]
```diff
@@ -1,22 +1,156 @@
...
+@pytest.mark.parametrize(
+    "asset_name,count", [("BTC", 1), ("BTC", 10), ("ETH", 100), ("DOT1", 1000)]
+)
+def test_buy_asset_from_market(
+    auth_broker, broker_account, assets_list, asset_name, count
+):
     market_name = "Yahoo"
-    asset_name = "BTC"
-    broker_account.cash_balance = decimal.Decimal("100000.0000")
+    asset_name = asset_name
```
Comment (Vladgasik) [inline cryptostock/market/tests/test_buy_asset_from_market.py:20]: I think it's redundant

**Review by Vladgasik — APPROVED** [review]

File: cryptostock/market/tests/test_buy_asset_from_market.py (hunk 1 above)
Comment (Lex-23) [reply cryptostock/market/tests/test_buy_asset_from_market.py:20]: I think, you are wrong. It is necessary for this test

Comment (vadim-denisov) [reply cryptostock/market/tests/test_buy_asset_from_market.py:20]: Why is it necessary?

Comment (Lex-23) [reply cryptostock/market/tests/test_buy_asset_from_market.py:20]: I wrongly understood Vlad. It is redundant

**Pull request merged**